content/reference-transaction-format.md

```diff
@@ -1049,6 +1049,7 @@ These codes indicate an error in the local server processing the transaction; it
 | telBAD\_DOMAIN | The transaction specified a domain value (for example, the `Domain` field of an [AccountSet transaction](#accountset)) that cannot be used, probably because it is too long to store in the ledger. |
 | telBAD\_PATH_COUNT | The transaction contains too many paths for the local server to process. |
 | telBAD\_PUBLIC\_KEY | The transaction specified a public key value (for example, as the `MessageKey` field of an [AccountSet transaction](#accountset)) that cannot be used, probably because it is too long. |
+| telCAN\_NOT\_QUEUE | The transaction did not meet the [open ledger cost](concept-transaction-cost.html), but this server did not queue it because it failed one of the server's heuristics for being "likely to succeed". You can try again later or sign and submit a replacement transaction with a higher transaction cost in the `Fee` field. |
```

Comment:
Most of those cases are for transactions which are not for the next sequence for the account. If they didn't get `telCAN_NOT_QUEUE`, they'd get `terPRE_SEQ` or some other error. Any exceptions to those rules (ie. if I wasn't detailed enough), where the transaction is the next one for the account should return `telINSUFF_FEE`.

Example:
Account's next sequence number is 100. Fee is escalated. Transaction 100 pays 10 drops, gets `terQUEUED`. Transaction 101 would affect the ability to claim a fee, gets `terQUEUED`. Transaction 102 gets `telCAN_NOT_QUEUE` pretty much no matter what, not because it doesn't have a high enough fee, but because transaction 101 is preventing it from getting queued and relayed. Once 101 gets into the open ledger, 102 may or may not also be able to get into the open ledger or be queued itself.

Another example:
Account's next sequence number is 100. Current ledger sequence is 500. Fee is escalated. Transaction 100 pays 10 drops, has LastLedgerSequence == 501. That gets `telINSUFF_FEE`. User resubmits transaction 100 with the open ledger fee, it succeeds.

Another:
Same scenario. Transaction 100 pays 10 drops, has LastLedgerSequence == 503, gets `terQUEUED`. Transaction 101 pays the open ledger fee, has LastLedgerSequence == 501. That'll get `telCAN_NOT_QUEUE`. Again, no matter how high the fee on 101, it can't get into the open ledger, because 100 is in the queue, and it can't get into the queue because the LastLedgerSequence is too low.